dash/dash.py

```diff
@@ -949,6 +986,12 @@ def get_asset_url(self, path):
             self.config.routes_pathname_prefix,
             path)
 
+    def _on_assets_change(self, _):
```

Comment:
Why pass the argument here? Can't we just call `on_change()`

Reply:
The watch function I got here is from another project of mine that I adapted to run on threads instead of asyncio. When a file changed in the other project, I would log the filename thus giving the changed filename as argument in the callback. But we don't have a logging system in-place so I just put a pass param. I guess I could remove the options we're not gonna use here.